packages/graphback-core/src/runtime/createInMemoryFilterPredicate.ts

```diff
@@ -1,3 +1,4 @@
+import { ObjectID } from 'mongodb';
```

Comment:
This is not related to this PR but wondering weather we should trim this to something like https://github.com/aerogear/graphback/compare/master...machi1990:chore/depends-on-bson-directly?expand=1 by having `bson` as a dependency and not mongodb as peerDependency. The idea will be that his can be used directly without requiring a user to have `mongodb` as depedency.

Reply:
You can cherry-pick the commit from my fork. Or merge this and I'll do it afterwards. Let me know what works for you.

Reply:
I will cherry pick yours, thanks for that.

Reply:
Cool.

Reply:
https://app.circleci.com/pipelines/github/aerogear/graphback/7175/workflows/35ccf632-8913-461e-baa0-837fc059a9ab/jobs/7315

I have cherry picked your commit but the build is failing - this is down to there being no ObjectId type declarations available from `bson-ext`, which are needed to creating the Scalar type [here](https://github.com/craicoverflow/graphback/blob/22a2be49ee8225adb88ce2c1ce55cdbe045197ae/packages/graphback-core/src/runtime/QueryFilter.ts#L14)